pkg/util/validation/exporter/ring.go

```diff
@@ -48,53 +45,30 @@ var ringOp = ring.NewOp([]ring.InstanceState{ring.ACTIVE, ring.LEAVING}, nil)
 
 // RingConfig holds the configuration for the overrides-exporter ring.
 type RingConfig struct {
-	Enabled bool `yaml:"enabled" category:"experimental"`
+	// Use common config shared with other components' ring config.
+	util.CommonRingConfig `yaml:",inline"`
 
-	// KV store details
-	KVStore          kv.Config     `yaml:"kvstore" doc:"description=The key-value store used to share the hash ring across multiple instances."`
-	HeartbeatPeriod  time.Duration `yaml:"heartbeat_period" category:"advanced"`
-	HeartbeatTimeout time.Duration `yaml:"heartbeat_timeout" category:"advanced"`
+	// Whether the ring is enabled for overrides-exporters.
+	Enabled bool `yaml:"enabled" category:"experimental"`
```

Comment:
[nit] Move this above `util.CommonRingConfig` so that in the auto-generated doc we show `enabled` as first entry.

Reply:
👍 will do